src/ng2/lazyLoadNgModule.ts

```diff
@@ -41,7 +41,7 @@ export type NgModuleToLoad = string | ModuleTypeCallback;
  * - Returns the new states array
  */
 export function loadNgModule(moduleToLoad: NgModuleToLoad): (transition: Transition) => Promise<LazyLoadResult> {
-  return function(transition: Transition) {
+  return (transition: Transition) => {
```

Comment:
To use `function` it should be exported, it will not work with AoT when it is passed to an NgModule